CMake error: file does not recognize sub-command COPY_FILE

With version v2023.03.0 and cmake 3.18.3 I get an errror when running CMake:

```
CMake Error at CMakeLists.txt:52 (file):
  file does not recognize sub-command COPY_FILE
```
That would probably require a newer version? Your website and CMakeLists,txt says 3.15 is sufficient

I can work around this one as it seems to be for WINDOWS anyway. But when compiling, it seems like you are not using the system python binary but something else that is not there: A python executable in the build directory.
```
....
-- Using Python Executable: /home/mblatt/src/dune/ResInsight-build/python
...
[  5%] Built target cafAnimControl
Scanning dependencies of target PipInstall
/bin/sh: 1: ../python: not found
make[2]: *** [GrpcInterface/CMakeFiles/PipInstall.dir/build.make:76: PipInstall] Fehler 127
make[1]: *** [CMakeFiles/Makefile2:2565: GrpcInterface/CMakeFiles/PipInstall.dir/all] Fehler 2
make[1]: *** Es wird auf noch nicht beendete Prozesse gewartet....
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,20 +45,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 # ##############################################################################
-# OPTIONAL: Configuration of ccache for MSVC
-# ##############################################################################
-find_program(ccache_exe ccache)
-if(ccache_exe)
-  file(COPY_FILE ${ccache_exe} ${CMAKE_BINARY_DIR}/cl.exe ONLY_IF_DIFFERENT)
-
-  set(CMAKE_VS_GLOBALS
-      "CLToolExe=cl.exe" "CLToolPath=${CMAKE_BINARY_DIR}"
-      "TrackFileAccess=false" "UseMultiToolTask=true"
-      "DebugInformationFormat=OldStyle"
-  )
-endif()
-
-# ##############################################################################
 # Setup the main platform defines
 # ##############################################################################
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")

